cmd/bosun/conf/conf.go

```diff
@@ -204,8 +203,7 @@ func errRecover(errp *error) {
 }
 
 type ConfItem struct {
```

Comment:
How about we delete this type entirely? It's not used. Unless we have a reason to keep around the original section text, we can just discard it.